website/src/components/canvas/options-bar/OptionsBar.jsx

```diff
@@ -152,6 +152,8 @@ class OptionsBar extends React.Component {
             <ShareCanvasModal />
             { userDivs }
           </span>
+          <ArchiveCanvas
+            canvasId={this.props.currentCanvas} />
```

Comment:
you should pass currentCanvas (which is just the id) here instead of the full canvas